<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>X10 Remote Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_ServerAddr">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_Connect">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_Exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_ServerPort">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>29876</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Host IP Address</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Host Port</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox_RemoteServer">
      <property name="text">
       <string>Remote Server</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>X10 Controls</string>
       </attribute>
       <widget class="QDial" name="dial_HouseCode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>101</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>House Code</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_UnitCode">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QDial" name="dial">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Unit Code</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_HouseCode">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>178</width>
          <height>29</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_Off">
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_On">
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Manual Commands</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_CmdText">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>341</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_SendManualCmd">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>81</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_CmdOutput">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>431</width>
          <height>201</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Clear">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>250</y>
          <width>86</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Clear</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_CmdOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_CmdText</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_SendManualCmd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_UnitCode</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_RemoteServer</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_Connect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_RemoteServer</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_ServerPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_RemoteServer</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_ServerAddr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_RemoteServer</sender>
   <signal>clicked(bool)</signal>
   <receiver>tabWidget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
